sonar-application/src/main/assembly/conf/sonar.properties

```diff
@@ -17,9 +17,7 @@
 #----- Embedded Database (default)
 # H2 embedded database server listening port, defaults to 9092
 #sonar.embeddedDatabase.port=9092
-
-
-#----- MySQL 5.x
+#----- MySQL 5.6 or greater
```

Comment:
you could associate this commit to SONAR-7260 (ie. a recent JIRA I found related to min version of supported DBs)